packages/frontend/components/Video.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import ReactPlayer from 'react-player'
+import styles from '../styles/videoBoard.module.scss'
+
+export interface VideoInterface {
+  id: string
+  title: string
+  creator: string
+  url: string
+}
+
+interface VideoProps {
+  key: string
+  video: VideoInterface
+}
+
+export default function Video(props: VideoProps) {
```

Comment:
you can destructure the prop like this:
```suggestion
export default function Video({ video }: VideoProps) {
```
you also need to change the things below like this then:
`props.video.title` -> `video.title` etc.

Reply:
destructuring seems really handy, fixed